cri/v1alpha2/cri_utils_test.go

```diff
@@ -785,7 +788,39 @@ func Test_imageToCriImage(t *testing.T) {
 		want    *runtime.Image
 		wantErr bool
 	}{
-	// TODO: Add test cases.
+		{
+			name: "normal test",
+			args: args{
+				image: &apitypes.ImageInfo{
+					ID:          "image-id",
+					RepoTags:    repoDigests,
+					RepoDigests: repoDigests,
+					Size:        1024,
+					Config: &apitypes.ContainerConfig{
+						User:    "foo",
+						Volumes: map[string]interface{}{"foo": "foo"},
+					},
+				},
+			},
+			want: &runtime.Image{
+				Id:          "image-id",
+				RepoTags:    repoDigests,
+				RepoDigests: repoDigests,
+				Size_:       uint64(1024),
+				Uid:         &runtime.Int64Value{},
+				Username:    "foo",
+				Volumes:     parseVolumesFromPouch(map[string]interface{}{"foo": "foo"}),
+			},
+			wantErr: nil,
```

Comment:
The type of `wantErr` is `bool`, we could not assign nil to it.

CI has found this mistake but not failed.